plugins/woocommerce/src/Blocks/AssetsController.php

```diff
@@ -196,6 +198,50 @@ private function get_prerender_resource_hints() {
 		return $urls;
 	}
 
+	/**
+	 * Get the block asset resource hints in the cache or null if not found.
+	 *
+	 * @return array|null Array of resource hints.
+	 */
+	private function get_block_asset_resource_hints_cache() {
+		if ( wp_is_development_mode( 'plugin' ) ) {
```

Comment:
Not seen this before 👍🏻

Reply:
It was my first time using this as well.  [Relatively new and introduced in 6.3](https://make.wordpress.org/core/2023/07/14/configuring-development-mode-in-6-3/).

It avoids some of the ambiguity around `SCRIPT_DEBUG` and `WP_DEBUG` that we previously used in development.